src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
```

Comment:
Some code in this PR are unrelated changes. Remember to branch out from `master` every time you start a new LO, and create PRs independently.